src/ecalc/libraries/neqsim/neqsim_ecalc_wrapper/thermo.py

```diff
@@ -43,6 +73,39 @@ def __init__(
             get_GERG2008_properties(thermodynamic_system=thermodynamic_system) if self._use_gerg else None
         )
 
+    def __str__(self) -> str:
+        """
+        An attempt to jsonify a dump of the internal NeqSim state for a given composition and properties.
+        :return: an unformatted, but somehow structured json object
+        """
+        fluid_state: Dict[str, List] = {}
```

Comment:
```suggestion
        fluid_state: Dict[str, List] = {}
```

Why first create a Dict, then create the pydantic model?